docs/Analyzer Configuration.md

```diff
@@ -66,6 +66,31 @@ Default Value: All output kinds
 
 Example: `dotnet_code_quality.CA2007.output_kind = ConsoleApplication, DynamicallyLinkedLibrary`
 
+### Required modifiers for analyzed APIs
```

Comment:
@gewarren Would you be able to do a quick review of the new option names? I will soon file a tracking bug in the docs repo for all the new options since https://docs.microsoft.com/en-us/visualstudio/code-quality/fxcop-analyzer-options?view=vs-2019 was last updated. Thanks!

Reply:
@mavasani They look good to me!

Reply:
Thanks!